src/pybind/mgr/dashboard/frontend/src/app/shared/services/task-list.service.ts

```diff
@@ -57,11 +57,9 @@ export class TaskListService implements OnDestroy {
     this.builders = builders || {};
 
     this.summaryDataSubscription = this.summaryService.subscribe((tasks: any) => {
-      if (tasks) {
-        this.getUpdate().subscribe((resp: any) => {
-          this.updateData(resp, tasks.executing_tasks.filter(this.taskFilter));
-        }, this.onFetchError);
-      }
+      this.getUpdate().subscribe((resp: any) => {
```

Comment:
Now that ```if``` condition has been removed we can set the tasks type as object:
```this.summaryService.subscribe((tasks: object)```.

Reply:
Done.